src/xdocs/google_style.xml

```diff
@@ -1485,6 +1501,35 @@
                     test</a>
                 </td>
               </tr>
+              <tr>
+                <td><a name="4.8.5.5"/>
+                  <a href="#4.8.5.5">
+                    <span class="wrapper inline">
+                      <img src="images/anchor.png" alt=""/>
+                    </span>
+                  </a>
+                </td>
+                <td>
+                  <a href="styleguides/google-java-style-20220203/javaguide.html#s4.8.5.5-local-parameter-annotation-style  ">
+                    4.8.5.5 Parameter and local variable annotations
+                  </a>
+                </td>
+                <td>
+                  <span class="wrapper inline">
+                    <img
+                        src="images/ban_red.png"
```

Comment:
Please use `--`
And wording `See 4.8.5.1 Type-use annotations`



"--" - There is no rule in this paragraph.

This chapter has no.rules, and just just reference to another chapter

Reply:
done

https://github.com/checkstyle/checkstyle/blob/5374c9edcc7df6d5e3cebb6b23e3577bd2179bcc/src/xdocs/google_style.xml#L1517-L1522